python/mmSolver/tools/channelsen/lib.py

```diff
@@ -0,0 +1,66 @@
+"""
+The Channel Sensitivitiy tool - user facing.
+"""
+
+import maya.cmds
+import mmSolver.logger
+import maya.mel
+
+
+LOG = mmSolver.logger.get_logger()
+
+
+def get_value():
+    """
+    Get the current channel sensitivity value.
+
+    :return: Current channel sensitivity value
+    :rtype: float
+    """
+    global_variable = __channelbox_global_variable()
+    current_value = maya.cmds.channelBox(global_variable,
+                                         speed=True,
+                                         query=True)
+    return current_value
+
+
+def set_value(sensitivity):
+    """
+    Set channel sensitivity value
+    :param sensitivity: A possible value to set channel sensitivity
+    value
+    :type: str or float or int
+    :return: None
+    """
+    current_value = get_value()
+    global_variable = __channelbox_global_variable()
+    cmd = 'channelBoxSettings useManips 1;'
+    maya.mel.eval(cmd)
+    maya.cmds.channelBox(global_variable,
+                         speed=current_value*sensitivity,
```

Comment:
Just set the value directly passed into the "set_value" function, rather than adding decrement/increment logic into the lib function.